cmake/Modules/Platform/wasi.cmake

```diff
@@ -0,0 +1,9 @@
+set(WASI 1)
+# NOTE: this is being defined *not* because WASI is unix-esque, which it is not.
```

Comment:
Is this file needed/used?

Reply:
Yes, it is needed, and used.  Without this, this controls the behaviour of the build.  See the example command in the commit message.

Reply:
Actually, it seems that with a bit more tweaking, I can get rid of it.